src/pages/PlanPage/subRoutes/CancelPlanPage/subRoutes/DowngradePlan/CancelButton/utils.js

```diff
@@ -8,17 +8,22 @@ export function getEndPeriod(unixPeriodEnd) {
 }
 
 export function loadBaremetrics() {
-  return new Promise((resolve) => {
-    if (window.barecancel && window.barecancel.created) {
```

Comment:
I didn't realize til this morning that we were incorrectly resolving this promise when we shouldn't have been because this attribute _was_ in the DOM but didn't actually mean the script was "working"